sdk/reservations/arm-reservations/README.md

```diff
@@ -34,9 +34,12 @@ import { AzureReservationAPI, AzureReservationAPIModels, AzureReservationAPIMapp
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
```

Comment:
subscriptionId is not required now as you have removed it below.

Reply:
Yes, I will fix it